slice.go

```diff
@@ -378,6 +378,17 @@ func SliceToMap[T any, K comparable, V any](collection []T, transform func(item
 	return Associate(collection, transform)
 }
 
+// SliceToSet returns a map with each unique element of the slice as a key.
+func SliceToSet[T comparable, Slice ~[]T](collection Slice) map[T]struct{} {
```

Comment:
Can you add this function in the Readme.

Reply:
Sure, I fixed it! Please review it again.